Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -126,6 +126,7 @@ class HomeFragmentViewModel @AssistedInject constructor(
         SubmissionRepository.deleteRegistrationToken()
```

Comment:
Maybe we should move all this logic to the `SubmissionRepository` to deduplicate the deletion logic

Reply:
Sounds reasonable, could we then also remove the static access and add a test for it?

Reply:
done, still some kind of static access to LocalData, will try to move to SubmissionSettings in an additional pr 👍